build.gradle

```diff
@@ -295,5 +295,14 @@ taskTree{
 
 defaultTasks 'ciBuild'
 
-preBuild.dependsOn includeDirBugFix
+apply from: "$rootDir/ktlint.gradle"
+
+afterEvaluate {
```

Comment:
do we need `afterEvaluate` here? Isn't the graph of tasks set up during the configuration phase anyway? IIRC, `dependsOn` is evaluated lazily so it's not necessary to do this in the `afterEvaluate` phase.

Also, since the `preBuild` task is going to be added by the plugins that are applied, we don't need to check for its existence because all the `dependsOn` declarations are not evaluated until the task graph is fully defined.

BUT

would it make sense to ignore those questions completely and just use one of the gradle plugins for ktlint instead? Such as:
https://github.com/jeremymailen/kotlinter-gradle
https://github.com/jlleitschuh/ktlint-gradle

Reply:
> would it make sense to ignore those questions completely and just use one of the gradle plugins for ktlint instead? Such as:

Not a fan of introducing 3rd party plugins for a plugin, they often are left unmaintained and deprecated.

I'll move the preBuild out.